### DISCUSSION TITLE
Sharing the extensions folder as well.

### DISCUSSION THREAD

**Discussion by nothingness6** [LykosAI/StabilityMatrix#712] [Ideas]
The one of the most useful feature of SM is sharing the model folder. But it doesn't support the extensions folder. The size of the each extensions folder is getting bigger.  

Plus, ComfyUI doesn't share the animate-diff and clip_vision folders, so I have to put the models in the folders separately. The size is pretty big. 

I hope this feature will be added in the next update. 
Cheers!
![Stability Matrix Issue-01](https://github.com/LykosAI/StabilityMatrix/assets/105210258/36af11d3-eeb9-4b51-8eb4-4b015c5d64ac)
![Stability Matrix Issue-02](https://github.com/LykosAI/StabilityMatrix/assets/105210258/e99f1c5f-1479-498f-86af-20fc92bcbd95)

**Comment by drdancm** [comment]
I would be very much interested in having this feature also.  Sharing extensions would be so very helpful because it would  also cut down on duplicate storage and install time requirements.

In any case a HUGE THANKS for creating and maintaining such an excellent Stability Matrix !
